unittests/autoconfig.py

```diff
@@ -56,6 +56,11 @@ class cxx_parsers_cfg(object):
         if 'msvc9' == gccxml.compiler:
             gccxml.define_symbols.append('_HAS_TR1=0')
 
+if cxx_parsers_cfg.gccxml.xml_generator:
```

Comment:
I don't think this change actually affects anything, but I was confused at first to see `Gccxml` being printed despite setting `xml_generator_path` and `xml_generator` to `caslxml` in `xml_generator.cfg`